Energy Saver/Pages/Statistics.cshtml

```diff
@@ -6,11 +6,54 @@
 }
 
 <h1 class="text-color">Statistics</h1>
-<div><canvas id="lineChart"></canvas></div>
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-lg-2">
+            <form asp-page-handler="year" method="post">
+                <div class="form-group">
+                    @Html.AntiForgeryToken()
+                    <select class="form-control" name="Year" id="yearDropdown" onchange="editYear()">
+                        @for (int i = DateTime.Now.Year; i > 1919; i--)
```

Comment:
you could make year choices dynamic, depending on whether that year has an entry